readme.md

```diff
@@ -127,18 +136,22 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** What are the arguments for `Model.updateOne()`?
 
 * **Your Answer:**
+The arguments are two objects. The first is a key:value pair that looks for a match in the database that has the same key:value values. The second object is another key:value pair which finds the key in that object found by the first object parameter and updates the value of that key to be the value given in this parameter. i.e. Series.updateOne({title: 'Twin Peaks'}, {start_year: 2018}) will find the series with the title of 'Twin Peaks' and update the start_year to be 2018.
```

Comment:
Correct! Just to add though, you can add more than one key-value pair as part of the `filter` argument.